models/extensionRequests.js

```diff
@@ -0,0 +1,88 @@
+const firestore = require("../utils/firestore");
+const extensionRequestsModel = firestore.collection("extensionRequests");
+const { buildExtensionRequests, formatExtensionRequest } = require("../utils/extensionRequests");
+
+/**
+ * Create Extension Request
+ * @param extensionRequestData { Object }: Body of the extension request
+ */
+const createExtensionRequest = async (extensionRequestData) => {
+  try {
+    const request = {
+      timestamp: (new Date().getTime() / 1000).toFixed(0),
```

Comment:
What are we trying to store here?
If I calculate the value (at the time of making this comment) it gives me `'1673074694'` which is: 
1. type of string 
2. If I do `new Date(1673074694)`  the corresponding output is `Tue Jan 20 1970 14:14:34`.

Is this what we want to store? 
Am I missing something here?